src/pages/likedVideoPage/likedVideoPage.js

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import { useLikedVideoContext } from "../../context/likedVideoContext/likedVideoContext";
+import { useSingleVideo } from "../../context/singleVideoContext/singleVideoContext";
```

Comment:
suggestion: export all contexts from the index.js file and import contexts here from the index.js file created in the context folder.